demos/imagenet_classification_demo/README.md

```diff
@@ -0,0 +1,122 @@
+    -nireq "<integer>"        Optional. Number of infer requests.
+    -nt "<integer>"           Optional. Number of top results. Default value is 5. Must be >= 1.
+    -res "<WxH>"              Optional. Set image grid resolution in format WxH. Default value is 1920x1080.
+    -no_show                  Optional. Disable showing of processed images.
+    -time "<integer>"         Optional. Time in seconds to execute program. Default is -1 (infinite time).
+    -u                        Optional. List of monitors to show initially.
+```
+
+The number of infer requests can be set by "-nireq" flag.
+* small nireq: low FPS (device cores aren't fully loaded) and low latency (images don't wait for long before being sent
+for inference).
+* nireq = number of cores: high FPS (full device utilization), median latency (images wait for some time). Considered optimal.
+* nireq > number of cores: high FPS (same as in the previous case), high latency (images wait for long time).
+
+Running the application with the empty list of options yields an error message.
+
+To run the demo, you can use public or pre-trained models. To download the pre-trained models, use the OpenVINO [Model Downloader](../../tools/downloader/README.md) or go to [https://download.01.org/opencv/](https://download.01.org/opencv/).
+
+> **NOTE**: Before running the demo with a trained model, make sure the model is converted to the Inference Engine format (\*.xml + \*.bin) using the [Model Optimizer tool](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html).
+
+For example, use the following command line command to run the application:
+```sh
+./imagenet_classification_demo -m <path_to_classification_model> \
+                      -i <path_to_folder_with_images> \
+                      -labels <path_to_file_with_list_of_labels> \
+                      -gt <path_to_file_with_list_of_classes> \
+                      -u CDM
+```
+
+## Demo Output
+
+The demo uses OpenCV to display the resulting image grid with classification results presented as a text above images.
+After the completion, it prints average telemetry values to the console.
+
+## Required files
+
+If you want to see classification results, you must use "-gt" and "-labels" flags to specify two .txt files
+containing lists of classes and labels.
+
+"Classes" file is used for matching image file names with right object classes.
+It has the following format:
+
+./ILSVRC2012_val_00000001.JPEG 65
+
+./ILSVRC2012_val_00000002.JPEG 970
+
+./ILSVRC2012_val_00000003.JPEG 230
+
+...
+
+"Labels" file contains the list of human-readable labels, one line for each class.
+You can find this file in source code: /open_model_zoo/demos/imagenet_classification_demos/synset_words.txt
+It has the following format:
+
+n01440764 tench, Tinca tinca
+
+n01443537 goldfish, Carassius auratus
+
+n01484850 great white shark, white shark, man-eater, man-eating shark, Carcharodon carcharias
```

Comment:
So why was that format chosen?

Reply:
@Wovchena 
There are line-breaks just for better look with Markdown formatting of this README file.

The actual file is like this:
n01440764 tench, Tinca tinca
n01443537 goldfish, Carassius auratus
n01484850 great white shark, white shark, man-eater, man-eating shark, Carcharodon carcharias

Katya told me that this format is the most common, as far as she knows.
It's also possible to use this format:
n12345678 1 word1, word2
n12345678 2 word3, word4
but these additional numbers don't contain any useful information, so I think they're not needed.

Reply:
Ok.
(The question was about including or not synset words.)